frontend/packages/pipelines-plugin/src/models/pipelines.ts

```diff
@@ -145,7 +145,6 @@ export const TriggerBindingModel: K8sKind = {
   id: 'triggerbinding',
   labelPlural: 'TriggerBindings',
   crd: true,
-  badge: BadgeType.TECH,
```

Comment:
should we keep this for any future use?

Reply:
Moving forward, we no longer need this badges for both pipeline and trigger components.